sdk/test/integration/global_tracer_configurations_test.rb

```diff
@@ -38,6 +38,12 @@
       it 'are all SpanData' do
         _(finished_spans.collect(&:class).uniq).must_equal([sdk::Trace::SpanData])
       end
+
+      it 'are all SpanData which have trace_flags and tracestate span context' do
+        finish_span_keys = finished_spans.collect(&:members).flatten.uniq
+        _(finish_span_keys).must_include(:tracestate)
+        _(finish_span_keys).must_include(:trace_flags)
+      end
```

Comment:
I think this test is overkill, since the previous one ensures all the finished spans are `SpanData`.

Reply:
Fair enough, I can remove this and work on adding a more robust end-to-end tracestate test that starts a span with a non-nil tracestate context, finishes+processes it, and verifies it is rec'd by the exporter with matching tracestate